<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseParserWidget</class>
 <widget class="QWidget" name="BaseParserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>825</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="pbOpen">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lePath"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Тип связи</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата
Длительность</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сеть
Базовая станция</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Lac
Cid</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Частота</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Долгота</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Широта</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Источник</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
